pkg/virt-handler/vm.go

```diff
@@ -1550,16 +1550,19 @@ func (d *VirtualMachineController) defaultExecute(key string,
 	// set true to ensure that no updates to the current VirtualMachineInstance state will occur
 	forceIgnoreSync := false
 
-	log.Log.Infof("Processing event %v", key)
-	if vmiExists {
-		log.Log.Object(vmi).Infof("VMI is in phase: %v\n", vmi.Status.Phase)
-	} else {
-		log.Log.Info("VMI does not exist")
-	}
-	if domainExists {
-		log.Log.Object(domain).Infof("Domain status: %v, reason: %v\n", domain.Status.Status, domain.Status.Reason)
+	log.Log.V(3).Infof("Processing event %v", key)
+
+	if vmiExists && domainExists {
+		log.Log.
+			ObjectWithPrefix("vmi", vmi).
```

Comment:
I see why a `ObjectWithPrefix` can be useful on one hand, on the other hand creating new keys does not seem like it makes it easy to search for the logs. In general i wonder how important logging the `Domain` is. It may have been a mistake in the first place to log the `Domain` ever. We should be able to construct a VMI reference out of the domain which we should probably log (even if the VMI does not exist anymore, but it still belongs to this VMI).

Reply:
Yes, in principle something like this should do it:


```
NewVMIReferenceWithUUID(domain.Name, domain.Namespace, domain.Spec.Metadata.KubeVirt.UID)
```

The returned object can then be logged.

Reply:
Thanks! Changed :)